docs/conf.py

```diff
@@ -1,5 +1,15 @@
+import os
+import sys
+
 import sphinx_rtd_theme
 
+cwd = os.getcwd()
+project_root = os.path.dirname(cwd)
+sys.path.insert(0, project_root)
+
+import cryptoconditions     # noqa
```

Comment:
I'm not sure.

If you do it the same way you do it in `setup.py` at least we will be consistent